src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Shape {
+    private double side;
+
+    public Square(String color, double side) {
+        super(Colors.valueOf(color));
```

Comment:
The `Colors.valueOf(color)` assumes the color string is always a valid enum constant. Consider adding error handling to manage cases where the color might not be valid.